lib/cc/cli/prepare/quality.rb

```diff
@@ -0,0 +1,59 @@
+require "safe_yaml/load"
+
+module CC
+  module CLI
+    class Prepare
+      class Quality < Command
+        ENGINES_CONFIG = {
+          "complexity-ruby" => {
+            "enabled" => true,
+            "channel" => "beta",
+          },
+          "duplication" => {
+            "enabled" => true,
+            "channel" => "cronopio",
+          },
+        }.freeze
+
+        def execute
+          Dir.chdir(CC::Analyzer::MountedPath.code.container_path) do
+            content =
+              if (existing_contents = read_codeclimate_yml)
+                modify(existing_contents)
+              else
+                { "engines" => ENGINES_CONFIG }
+              end
+
+            write_codeclimate_yml(content)
+          end
+        end
+
+        private
+
+        def modify(content)
```

Comment:
Actually, going to not do that for now. From the out-side the method is meant to take and return something. The fact this it uses mutation is kind of a quirk of the current set of changes we need to make right now. We can swing one way or the other in the future, but I'll keep the name more aligned with how it's used (non-mutating).